resources/charlie/chainspec.toml

```diff
@@ -44,7 +44,10 @@ entropy_duration_millis = 10800000
 # Keep voting on the switch block for a fixed amount of time; effective if the summit level is zero.
 # 172800000 ms = 2 days.
 voting_period_duration_millis = 172800000
-# Integer between 0 and 99 representing the fault tolerance threshold as a percentage, used by the internal finalizer.
+# Integer between 1 and 99 representing the fault tolerance threshold as a percentage, used by the internal finalizer.
+# It is the percentage of validators that would need to be malicious to make two honest nodes see two conflicting
+# blocks as finalized: A higher value F makes it safer to rely on finalized blocks. It also makes it more difficult to
+# finalize blocks, however, and requires at least 50% - F/2 validators to be working correctly.
```

Comment:
Shouldn't it be `50% + F/2` ? Currently, Charlie has FTT set to 50 and you said it needs 75% of validators to finalize.

Reply:
Ah, good catch! :man_facepalming: